app/templates/src/static/css/cf-theme-overrides.less

```diff
@@ -137,6 +137,12 @@
 // .form-label-helper-text
 @label-helper:                  @darkgray;
 
+// .form-l-inset_container
+@input-inset-bg:                @gray-20;
+
+// .form-l-inset_container.is-checked
```

Comment:
Why are we styling them at all for IE8? We ought to be enhancing the browsers that support it, and let those that don't use default radios/checkboxes.